### DISCUSSION TITLE
TSA and Change, Aftereffect, Trend Analysis

### DISCUSSION THREAD

**Discussion by prenner-eo** [davidfrantz/force#105] [Q&A] [answered]
Hey all,

I am currently working on a Time Series Analysis. I would like to run an NDVI-based change-aftereffect-trend analysis. The NDVI images in the time series also look as expected, but the spectral temporal metrics in the data are already output with zero values, except for the number of pixels. For the processing with folding parameters a segmentation error appears and for the SPLITS-based LSPs only empty tiffs are output.
Here are some of the NDVI plots:

![ndvi_auszug_1998-2020](https://user-images.githubusercontent.com/50298512/121914039-33707900-cd32-11eb-8d45-185fdf9ab419.gif)

Attached is my parameter file, I had to rename it to .txt in order to upload it here. I already tried multiple changes to perform the CAT-analysis.
[TSA_bbr_ndvi.txt](https://github.com/davidfrantz/force/files/6649390/TSA_bbr_ndvi.txt)
My goal for now is to get an output like the following example from the FORCE documentation:

![image](https://user-images.githubusercontent.com/50298512/121915674-b7773080-cd33-11eb-8f79-94be6535803b.png)

Both the tutorial (https://force-eo.readthedocs.io/en/latest/howto/stm.html) and the documentation (https://force-eo.readthedocs.io/en/latest/components/higher-level/tsa/index.html) could not really help me yet.
I hope there is someone here with more experience in doing these kind of analysis and could help me out a bit.

Best regards 
Paul

**Comment by davidfrantz** [comment, marked as answer]
Hi @BlackeBlack ,
which version are you using? Can you please attach your parameter file?
Best,
David

**Reply by prenner-eo** [reply]
Hey David,
the parameter file is attached as TSA_bbr_ndvi.txt I cant upload it as .prm. I use FORCE on version 3.6.5 with Singularity.
Best
Paul

**Reply by davidfrantz** [reply]
Hi Paul, 
this is probably directly related to this issue here: https://github.com/davidfrantz/force/issues/75
Please try with the develop version (I didn't find time to prepare a new major release yet..)
Best,
David

**Reply by prenner-eo** [reply]
It seems that the processing works so far. Thanks for your help.
